### PR TITLE
chore: updated bug report template

### PR DESCRIPTION
**Pull request by 10ko**
Updated the Bug Report template to include the `garden get debug-info` command.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -40,3 +40,6 @@ about: Create a report to help make garden better
 `kubectl version`
 
 `docker version`
+
+<!-- Please run the get debug-info command and attach the output zip file -->
```
Comment (eysi09) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: Could you add quotes or back ticks around the "get debug-info" part?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -40,3 +40,6 @@ about: Create a report to help make garden better
 `kubectl version`
 
 `docker version`
```
Comment (edvald) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: I think we should also remove both the docker and kubectl version checks. They're no longer needed.

**Comment by eysi09** [comment]
I went ahead and removed the `kubectl` and `docker` version requests from the template.

**Comment by thsig** [comment]
Merged.

**Pull request closed without merging**